hbase_regionserver/_metrics.yml

```diff
@@ -41,14 +41,17 @@
   attribute: sink.appliedOps
   metric_type: gauge
   alias: hbase.regionserver.replication.sink.applied_ops
+  description: Number of WAL entries applied on replication sink.
 - bean: Hadoop:service=HBase,name=RegionServer,sub=Replication
```

Comment:
sink.appliedOps is not latency but number of operations. There seems to be no similar convention in existing entries. "operation" could be a candidate. Do you have any preference?